### PR TITLE
docs: missing references in visualization

### PR DESCRIPTION
**Pull request by SmiteDeluxe**
### Summary of Changes

Added some missing references in documentation.

### CODE REVIEW

File: docs/Stdlib/python/Tutorials/visualization.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ We can infer that tickets for "First Class"-rooms (Pclass = 1.0) are much more e
 Also, the difference between second and third is less pronounced.
 
 ## Other plots
-Some other plots that might be useful to understanding our data and relationships within it are boxplots, histograms and scatterplots.
+Some other plots that might be useful to understanding our data and relationships within it are [boxplots][safeds.plotting._boxplot], [histograms][safeds.plotting._histogram] and [scatterplots][safeds.plotting._scatterplot].
```
Comment (WinPlay02) [inline docs/Stdlib/python/Tutorials/visualization.md:76]: I think we should not reference internal names in the docs

**Review by WinPlay02 — CHANGES_REQUESTED** [review]

File: docs/Stdlib/python/Tutorials/visualization.md (hunk 1 above)
Comment (WinPlay02) [inline docs/Stdlib/python/Tutorials/visualization.md:76]:
```suggestion
Some other plots that might be useful to understanding our data and relationships within it are [boxplots][safeds.plotting.boxplot], [histograms][safeds.plotting.histogram] and [scatterplots][safeds.plotting.scatterplot].
```

This should work as a replacement @SmiteDeluxe

Comment (SmiteDeluxe) [reply docs/Stdlib/python/Tutorials/visualization.md:76]: Will it? My suggestion would've been safeds.plotting._boxplot.boxplot
Will commit yours and see!

File: docs/Stdlib/python/Tutorials/visualization.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ correlation_heatmap(data_only_numerics)
 
 As you can see, the columns _Fare_ and _Pclass_ (Passenger Class) seem to heavily correlate. Let's have another look at that.
 ## Lineplot
-We'll use a lineplot to better understand their relationship, which is one way to visualise the relationship between two variables easily.
+We'll use a [lineplot][safeds.plotting._lineplot] to better understand their relationship, which is one way to visualise the relationship between two variables easily.
```
Comment (WinPlay02) [inline docs/Stdlib/python/Tutorials/visualization.md:61]:
```suggestion
We'll use a [lineplot][safeds.plotting.lineplot] to better understand their relationship, which is one way to visualise the relationship between two variables easily.
```

File: docs/Stdlib/python/Tutorials/visualization.md [hunk 3]
```diff
@@ -32,15 +32,15 @@ data.summary()  # returns a table with various statistics for each column
...
+this column. Since this isn't helpful for our usecase, we will [drop][safeds.data.tabular._table.Table.drop_columns] it.
 
 ```python
 data_cleaned = data.drop_columns(["PassengerId"])
 ```
 ## Heatmap
 Now we have a rough idea of what we are dealing with. But there is still a lot we do not know about the dataset.
 A good next step is to start plotting our columns against each other.
-One way to gain a quick understanding of how the different columns relate to each other is a Heatmap.
+One way to gain a quick understanding of how the different columns relate to each other is a [Heatmap][safeds.plotting._correlation_heatmap].
```
Comment (WinPlay02) [inline docs/Stdlib/python/Tutorials/visualization.md:43]:
```suggestion
One way to gain a quick understanding of how the different columns relate to each other is a [Heatmap][safeds.plotting.correlation_heatmap].
```

**Review by WinPlay02 — APPROVED** [review]

**Pull request merged**